packages/assets/README.md

```diff
@@ -79,27 +79,27 @@ Or:
 All specified handedness values must be present in the associated registry profile.  For example, if the registry has a defined `left-right` layout, the asset json can override `left` but not `none`.
 
 ### Layouts
-Within each layout, there must be a `rootNodeName`, `assetPath`, and/or a `components` property. The `rootNodeName` is the top node in the 3D asset hierarchy representing the motion controller, and is set to `<profile id>-<handedness>` by default. The `assetPath` is the relative path to the asset for the layout, and is set to `<handedness>.glb` by default.  The `components` property is explained further in the [components](#components) section.
+Within each layout, there must be a `rootNodeName`, `assetPath`, and/or a `components` property. The `rootNodeName` describes the top node in the 3D asset hierarchy representing the motion controller. As Maya files cannot name nodes with "-" characters, the default value for this node name is `<profile id>_<handedness>` with all "-" in the profile id changed to "_". The `assetPath` is the relative path to the asset for the layout, and is set to `<handedness>.glb` by default.  The `components` property is explained further in the [components](#components) section.
```

Comment:
Make sure this actually renders right in GitHub. In the review preview I'm seeing it detect the "_" as the start of italics. But I also know there's differences between how this view detects the markdown and how the markdown actually renders. If it's being problematic probably just surround it in backticks instead: `_`

Reply:
I can't seem to find where you're seeing that issue, but I just added the backticks.  Can you double check that the error you're seeing is gone?